Task-1/Khushi Jain/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+  <title>TEDx is Almost Here🚀</title>
+  <link rel="stylesheet" href="style.css" />
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet" />
+</head>
+<body>
+  <div class="animated-bg"></div>
+
+  <div class="container">
+    <h1>TEDx Countdown-2025🚨</h1>
+    <p id="event-time">TEDx VSSUT Presents:Rise Beyond✨</p>
+
+    <div class="countdown">
+      <div class="time-box">
+        <span id="days">00</span>
+        <small>Days</small>
+      </div>
+      <div class="time-box">
+        <span id="hours">00</span>
+        <small>Hours</small>
+      </div>
+      <div class="time-box">
+        <span id="minutes">00</span>
+        <small>Minutes</small>
+      </div>
+      <div class="time-box">
+        <span id="seconds">00</span>
+        <small>Seconds</small>
+      </div>
+    </div>
+
+    <div class="set-event">
+      <input type="datetime-local" id="event-time" />

```

Comment:
Youre using the same Id here, so the DOM will consider the above element as having the id, not this one. Youre using this id to get the time in script.js to update the event time. But as it is selecting the above element, the program isnt setting the event date and just showing the 'Please enter valid date' error